src/runtime/crt/host/microtvm_api_server.py

```diff
@@ -67,6 +80,27 @@ def server_info_query(self, tvm_version):
     # The build target given to make
     BUILD_TARGET = "build/main"
 
+    def _populate_makefile(
+        self,
+        makefile_template_path: pathlib.Path,
+        makefile_path: pathlib.Path,
+        memory_size: int,
+    ):
+        """Generate Makefile from template."""
+        flags = {
+            "MEMORY_SIZE_BYTES": str(memory_size),
+        }
+
+        regex = re.compile(r"([A-Z_]+) := (<[A-Z_]+>)")
+        with open(makefile_path, "w") as makefile_f:
+            with open(makefile_template_path, "r") as makefile_template_f:
+                for line in makefile_template_f:
+                    m = regex.match(line)
+                    if m:
+                        var, token = m.groups()
+                        line = re.sub(token, flags[var], line)
```

Comment:
Sorry, minutes after I've suggested using `re.sub` here I changed to use string's `replace()` -- you read the comment too fast! :-) I think `replace` is better because `token` is not really a regex so it's clearer, hence could you use:

```
line = line.replace(token, flags[var])
```

instead  of `re.sub` here, please?